packages/model-viewer/src/features/scene-graph/api.ts

```diff
@@ -39,7 +43,29 @@ export declare interface Model {
    * correspond to the listing of materials in the glTF, with the possible
    * addition of a default material at the end.
    */
-  readonly materials: Readonly<Array<Material>>;
+  readonly materials: Readonly<Material[]>;
+
+  /**
+   * Gets a material(s) by name.
+   * @param name the name of the material to return.
+   * @returns a list of materials who's name is equal to `name` or an empty list
+   *     if no matching materials found.
+   */
+  getMaterialByName(name: string): Material[];
```

Comment:
Should we just return  the first one instead? Seems more ergonomic, and I don't  really want  to encourage people to have multiple materials with the same name.

Reply:
The problem is that material names don't have to be unique in glTF so you could end up in this situation, but we can just specify that it returns the 'first' material that matches.